build.gradle

```diff
@@ -224,6 +224,36 @@ subprojects {
             }
         }
     }
+
+
+    task copyDependencies(type: Copy) {
+        from configurations.runtime
+        into new File(buildDir, "dex")
```

Comment:
`File(buildDir, "dex-all")` should be in some shared config and re-used by these tasks. I suggest something like `dex-all` just to distinguish from other invocations of `dex`